dev-python/pipenv/files/distable-networked-tests.patch

```diff
@@ -0,0 +1,41 @@
+:100644 100644 4b9cd75c 00000000 M	tests/unit/test_utils.py
+
+diff --git a/tests/unit/test_utils.py b/tests/unit/test_utils.py
+index 4b9cd75c..9f2dd043 100644
+--- a/tests/unit/test_utils.py
++++ b/tests/unit/test_utils.py
+@@ -47,16 +47,16 @@ DEP_PIP_PAIRS = [
+         },
+         "-e svn+svn://svn.myproject.org/svn/MyProject#egg=MyProject",
+     ),
+-    (
+-        # Extras in url
+-        {
+-            "discord.py": {
+-                "file": "https://github.com/Rapptz/discord.py/archive/async.zip",
+-                "extras": ["voice"],
+-            }
+-        },
+-        "https://github.com/Rapptz/discord.py/archive/async.zip#egg=discord.py[voice]",
+-    ),
++    #(
++    #    # Extras in url
++    #    {
++    #        "discord.py": {
++    #            "file": "https://github.com/Rapptz/discord.py/archive/async.zip",
++    #            "extras": ["voice"],
++    #        }
++    #    },
++    #    "https://github.com/Rapptz/discord.py/archive/async.zip#egg=discord.py[voice]",
++    #), This test case requires network and will fail in gentoo's sandbox
```

Comment:
Let's drop the code rather than commenting it out and put the comment "This test case requires network and will fail in gentoo's sandbox" into a commit message, e.g. using `git commit` and `git format-patch -1`. Thank you!

Reply:
Done.